docs/docs/sourcing-from-seams-cms.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Sourcing from Seams-CMS
+---
+
+## Overview
```

Comment:
```suggestion
```

You can remove this since the title of the page will serve as a heading for this section about Seams-CMS itself.